.github/workflows/JuliaNightly.yml

```
@@ -16,32 +16,34 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - 'nightly'
+          - "nightly"
         os:
           - ubuntu-latest
           - macOS-latest
           - windows-latest
         arch:
           - x64
-        # 32-bit Julia binaries are not available on macOS
+          - x86
         exclude:
+          # 32-bit Julia binaries are not available on macOS
           - os: macOS-latest
             arch: x86
     steps:
       - uses: actions/checkout@v2
       - uses: julia-actions/setup-julia@v1
         with:
           version: ${{ matrix.version }}
-          arch: ${{ matrix.arch }}
-      - uses: actions/cache@v1
+          arich: ${{ matrix.arch }}
```

I don't think this was meant to be changed.

```suggestion
          arch: ${{ matrix.arch }}
```